packages/ckeditor5-dev-docs/lib/index.js

```diff
@@ -5,103 +5,7 @@
 
 'use strict';
 
-const fs = require( 'fs-extra' );
-const tmp = require( 'tmp' );
-const glob = require( 'fast-glob' );
-
-const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
-
 module.exports = {
-	build
+	build: require( './docs/index' ),
+	verify: require( './web-crawler/index' )
```

Comment:
I am not sure about the name so that we can improve.

Reply:
The name is fine for me.